src/renderer/components/dock/dock.store.ts

```diff
@@ -35,16 +35,24 @@ export enum TabKind {
   POD_LOGS = "pod-logs",
 }
 
-export interface IDockTab {
-  id?: TabId;
+export interface DockTabDescriptor {
```

Comment:
I found new name a bit verbose then it should be. Why not use `DockTab`? `Descriptor` part here is like `Items`, `Object` and similar names that can be omitted.

Reply:
Sure I can change the name to `DockTab`

Reply:
I chose this name because `DockTab` is already a component, I agree `DockTabDescriptor` is a tad long.